code/datums/chatmessage.dm

```diff
@@ -81,6 +81,7 @@
 			LAZYREMOVEASSOC(owned_by.seen_messages, message_loc, src)
 		owned_by.images.Remove(message)
 
+	SSrunechat.message_queue -= src
```

Comment:
would prefer to have the datum remember the callback and remove IT here, to avoid extra cost in fire()